src/shared/NavComponent.jsx

```diff
@@ -88,7 +99,11 @@ export default function NavComponent(props) {
           </Nav>
         </Navbar.Collapse>
       </Navbar>
-      {!hideSeparator && <IconSeparator />}
+      {!hideSeparator && (
+        <div style={{ overflowX: "hidden" }}>
```

Comment:
@chiaberry @sergiogcx—@jgabitto and I had a philosophical conversation about when it's appropriate to use inline styles vs a stylesheet.  my take is that I prefer inline styles for one-off styling because it’s obvious when you’re reading the code, but only up until you need to reuse a style in multiple places. then, don’t repeat yourself and put it in a stylesheet.

we're curious how y'all feel about this, particularly in this line.

Reply:
I agree with your take, if its one-off styling and (only one attribute), inline styles are easier to read and follow.